src/clj/puppetlabs/http/client/async.clj

```diff
@@ -293,14 +294,36 @@
         :else
           (DefaultRedirectStrategy.))))
 
+(schema/defn request-config :- RequestConfig
```

Comment:
Oh duh, just realized it's private. Been doing functional so long I almost forgot about those!